charts/coredns/values.yaml

```
@@ -351,6 +351,8 @@ autoscaler:
     # pullSecrets:
     #   - name: myRegistryKeySecretName
 
+  lifecycle: ~
```

We should default to an empty object:

```suggestion
  lifecycle: {}
```